pkg/api/req.go

```diff
@@ -19,3 +19,11 @@ type PeerAddReq struct {
 type VolOptionReq struct {
 	Options map[string]string `json:"options"`
 }
+
+// VolExpandReq represents a request to expand the volume by adding more bricks
+type VolExpandReq struct {
```

Comment:
Also, please remove the copy of this struct from `commands/volumes/volume-expand.go`. Import it there.